sensor/kubernetes/main.go

```diff
@@ -24,6 +25,10 @@ var (
 	log = logging.LoggerForModule()
 )
 
+func init() {
+	memlimit.SetMemoryLimit()
```

Comment:
Why not run this in main? What do we get by having it in init?

Reply:
`init` runs prior to `main`, so this ensures that it is always (one of) the first things to run. I see it as a way to initialize even prior the main program runs